schema/metaschema/meta_schema_model.py

```diff
@@ -0,0 +1,288 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+from enum import Enum
+from typing import Annotated, Any, List, Literal
+
+from pydantic import BaseModel, ConfigDict, Field, RootModel, constr
+
+
+class FieldRef(RootModel):
+    root: str
+    (
+        "Reference to Sub Schema using JSON Schema resolver. "
+        "Example '#/keys/mykey' will resolve the schema for 'mykey' under the root dictionary of the current schema"
+    )
+
+    def __str__(self):
+        return self.root.__str__()
+
+
+class Deprecation(BaseModel):
+    warning: bool = True
+    "Emit deprecation warning if key is set"
```

Comment:
should it be docstring instead `"""` for docymenation ?

Reply:
fixing